src/Constants/Constants/AppVersion/index.ts

```diff
@@ -0,0 +1,2 @@
+export const GOOGLE_STORE_URL = "https://play.google.com/store/apps/details?id=org.vechain.veworld.app"
+export const APPLE_STORE_URL = (countryCode: string) => `https://apps.apple.com/${countryCode}/app/veworld/id6446854569`
```

Comment:
Is the country code needed for redirecting to the correct page
I just tried omitting it
`https://apps.apple.com/app/veworld/id6446854569`
And it auto redirects to the correct page on my side